diagnostic.md

```diff
@@ -36,13 +44,21 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --request GET http://10.13.105.206:3000/
```

Comment:
Missing `--include` and path. How do you think we might refer to a single donut (with an id of, say, 23)?